frontend/src/config.ts

```diff
@@ -46,31 +40,33 @@ export const SUPPORT_TELEGRAM = 'https://t.me/rizhaow';
 // Name
 export const MIN_NAME_LEN = 3;
 export const MAX_NAME_LEN = 30;
-export const REGEX_NAME = XRegExp(/^[\p{L}\p{M}\p{Zs}.'-]{3,30}$/u);
 /*
 Note: Name can only consist of uppercase & lowercase alphabets, 
 characters from other languages (e.g. Chinese, accented Latin alphabets), blank spaces
 as well as special characters dot(.), dash(-) and apostrophe('). No other special characters
 allowed.
 Name must also be between 3 to 30 characters. Update this regex when length limitations change
 */
+export const REGEX_NAME = XRegExp(/^[\p{L}\p{M}\p{Zs}.'-]{3,30}$/u);
 
-//Telegram Handle
+// Telegram Handle
 export const REGEX_TELEGRAM =
   /^(?=.{5,32}$)(?!.*__)(?!^(telegram|admin|support))[a-zA-Z][a-zA-Z0-9_]*[a-zA-Z0-9]$/;
 
-//Username
+// Username
 export const MIN_USERNAME_LEN = 3;
 export const MAX_USERNAME_LEN = 30;
-export const REGEX_USERNAME = /^([A-Za-z0-9]){3,30}$/;
 /*
 Note: username can only consist of uppercase & lowercase alphabets and numbers, 
 and must be between 3 to 30 characters. Update this regex when length limitations change
 */
+export const REGEX_USERNAME = /^([A-Za-z0-9]){3,30}$/;
```

Comment:
I see that we are using the regex for validating username length and username pattern

In that case, I wonder how do we control the validation message for each individual case.

Reply:
i did not introduce this change

Reply:
Can use Yup .min, .max for length validation & corresponding error message. The length validations in the regex is not strictly necessary, but just exists as a fallthrough.